docs/instrumentation-list.yaml

```diff
@@ -6859,12 +6859,45 @@ libraries:
         attributes: []
   jsp:
   - name: jsp-2.3
+    display_name: JSP (JavaServer Pages)
+    description: |
+      This instrumentation enables view spans for JSP page rendering and compilation (view spans are disabled by default).
```

Comment:
@trask the jsp compile spans are enabled by default. Should we disable them?

Reply:
👍